### DISCUSSION TITLE
Histoire's future after Storybook v7

### DISCUSSION THREAD

**Discussion by madeleineostoja** [histoire-dev/histoire#414] [General]
I'm evaluating Histoire as a potential replacement for Storybook in a fairly large monorepo that houses our whole platform and design system. We're considering migrating because of Histoire's great Svelte integration (our whole stack is svelte/sveltekit) and because of countless issues we've had getting storybook's heavy handed approach to work across the monorepo in the past.

Given the size of our project a migration is a big decision and Histoire is a pretty fresh project. Now that Storybook v7 has announced native vite support, including extending existing vite config, what does Histoire's future look like? From reading the "why Histoire" page in the docs it seems that native vite support is Histoire's big selling point. Will it continue to be maintained and evolved in the face of Storybook v7, or is it still at the experimental/exploratory stage?

Svelte stories with Histoire have great DX and the conifg is nice and light, but I'm concerned about Histoire sticking around if we were to migrate. Seeing one of the core Vue maintainers on the team is a good sign, but would be great to hear about roadmap and plans from the team. 

Thanks!

**Comment by madeleineostoja** [comment]
bump?

**Comment by ThomasKientz** [comment]
I would stick with Storybook if it's a large and long term project for maintenance reasons. Histoire have not reach v1 yet. It is developed by 1 person (from commits history of last month). Don't misunderstand me, Histoire is great and I love it, but it is still a 1 person shiny toy project for the moment.

**Reply by madeleineostoja** [reply]
Yeah that's how I feel as well, I was hoping that 1 person being on the Vue core team would carry some weight but it does feel like a side project still atm.

It's a bummer because I still run into frustrating issues with storybook constantly

**Comment by kiddicodes** [comment]
@Akryum Hi, sorry to ping you directly on this. But would really love having your input on this question 💭
